src/client/settings/Settings.js

```diff
@@ -101,46 +101,46 @@ export default class Settings extends React.Component {
   }
 
   languages = {
-    en: 'English',
-    id: 'Bahasa Indonesia - Indonesian',
-    ms: 'Bahasa Melayu - Malay',
-    ca: 'Català - Catalan',
-    cs: 'Čeština - Czech',
-    da: 'Dansk - Danish',
-    de: 'Deutsch - German',
-    et: 'Eesti - Estonian',
-    es: 'Español - Spanish',
-    fil: 'Filipino',
-    fr: 'Français - French',
-    hr: 'Hrvatski - Croatian',
-    it: 'Italiano - Italian',
-    hu: 'Magyar - Hungarian',
-    nl: 'Nederlands - Dutch',
-    no: 'Norsk - Norwegian',
-    pl: 'Polski - Polish',
-    pt: 'Português - Portuguese',
-    ro: 'Română - Romanian',
-    sl: 'Slovenščina - Slovenian',
-    sv: 'Svenska - Swedish',
-    vi: 'Tiếng Việt - Vietnamese',
-    tr: 'Türkçe - Turkish',
-    yo: 'Yorùbá - Yoruba',
-    el: 'Ελληνικά - Greek',
-    bg: 'Български език - Bulgarian',
-    ru: 'Русский - Russian',
-    uk: 'Українська мова - Ukrainian',
-    he: 'עִבְרִית - Hebrew',
-    ar: 'العربية - Arabic‏',
-    np: 'नेपाली - Nepali',
-    hi: 'हिन्दी - Hindi',
-    as: 'অসমীয়া - Assamese',
-    bn: 'বাংলা - Bengali',
-    ta: 'தமிழ் - Tamil',
-    lo: 'ພາສາລາວ - Lao',
-    th: 'ภาษาไทย - Thai',
-    ko: '한국어 - Korean',
-    ja: '日本語 - Japanese',
-    zh: '简体中文 - Simplified Chinese',
+    'ar-SA': 'العربية - Arabic‏',
```

Comment:
Can you keep the list in the exact same order than before? With English first

Reply:
Can we keep it alphabetized its easier to read and make changes to?

Reply:
yeah for settings list, i see, we can since this displays dropdown